amdahl/amdahl.py

```diff
@@ -110,19 +119,24 @@ def parse_command_line():
 def amdahl():
     """Amdahl's law illustrator (with fake work)"""
     rank = MPI.COMM_WORLD.Get_rank()
-    # Only the root process handles the command line arguments
+    # All processes handle (at least some) command line arguments
```

Comment:
There's a problem with this. It's not necessarily true that all nodes have _access_ to the command line arguments (IIRC it depends on the MPI runtime). A more robust approach is to always handle this on the root process and broadcast the `terse` option to the other processes.

Reply:
Working on this now.